docs/docs/08-working-with-the-browser.md

```diff
@@ -123,7 +123,7 @@ In addition to that philosophy, we've also taken the stance that we, as authors
 * `String.prototype.split`
 * `String.prototype.trim`
 
-`es5-sham.js`, also from [kriskowal's es5-shim](https://github.com/kriskowal/es5-shim), provides the following that React needs:
+`es5-sham.js`, also from [kriskowal's es5-shim](https://github.com/es-shims/es5-shim), provides the following that React needs:
```

Comment:
I thought about that, but even though I own and maintain it now, I saw no need to remove Kris' credit for creating it and building it up in the first place. These are your docs, so it's your call :-)